spec/models/social_networking/serializers/comment_serializer_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "spec_helper"
+
+module SocialNetworking
+  module Serializers
+    RSpec.describe CommentSerializer, type: :model do
```

Comment:
Why did you decide to make this a model spec? Also, why are you using fixtures as opposed to doubles?